tests/client/object_access_control.c

```diff
@@ -153,6 +153,7 @@ static uint8_t prv_read(uint16_t instanceId, int * numDataP,
         }
     }
 
+    ni = 0;
```

Comment:
Because "nothing" is intialized at definition.
May be something for the cleanup :-).